tests/ui-api-layer-acceptance-tests/domain/remoteenvironment/remoteenvironment_test.go

```diff
@@ -0,0 +1,114 @@
+// +build acceptance
+
+package remoteenvironment
+
+import (
+	"testing"
+	"github.com/kyma-project/kyma/tests/ui-api-layer-acceptance-tests/dex"
+	"github.com/kyma-project/kyma/tests/ui-api-layer-acceptance-tests/graphql"
+	"github.com/stretchr/testify/require"
+	"github.com/magiconair/properties/assert"
+)
+
+type reMutationResponse struct {
+	Name        string            `json:"name"`
+	Description string            `json:"description"`
+	Labels      map[string]string `json:"labels"`
+}
+
+type deleteReResponse struct {
```

Comment:
but delete and create are mutations, what's more, you are using this struct both for resp and req.
To be consistent name it `reDeleteMutation`